src/libmongoc/src/mongoc/mongoc-scram.c

```diff
@@ -1035,12 +1036,12 @@ _mongoc_sasl_prep_impl (const char *name,
                         int in_utf8_len,
```

Comment:
This parameter is now unused: should `_mongoc_utf8_string_length` account for the possibility of `in_utf8_len < strlen(in_utf8)`?

Reply:
I'm a little unsure on what you mean by this. In all cases of `_mongoc_sasl_prep_impl` being called, `in_utf8_len` is passed through as `strlen(in_utf8)`, meaning that `in_utf8_len == strlen(in_utf8)`.

Reply:
Good point. I suggest removing the redundant/unused parameter from `_mongoc_sasl_prep` and `_mongoc_sasl_prep_impl` to address the unused parameter warning(s).